.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+/out/
+/.idea/
+hw2.iml
```

Comment:
it is all okay, but I'd use a wildcard instead

e.g.`*.iml`

Reply:
Done